src/components/TodoList2.tsx

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
```

Comment:
Use a better name for this file instead of `TodoList2` 
<img width="368" alt="image" src="https://github.com/user-attachments/assets/410f53fe-bd6f-4918-8cdf-baf046a88f51" />